server/src/main/java/org/elasticsearch/transport/OutboundMessage.java

```diff
@@ -186,17 +182,7 @@ private static byte setStatus(boolean isHandshake, Writeable message) {
 
         @Override
         public String toString() {
-            return "Response{"
-                + requestId
-                + "}{"
-                + isError()
-                + "}{"
-                + isCompress()
-                + "}{"
-                + isHandshake()
-                + "}{"
-                + message.getClass()
```

Comment:
I'd rather we kept some way to identify the response type, although I must say I never much liked sharing its class here. Could we plumb in the `action` from the `TcpTransportChannel` down to here first?